src/lib/integration_test_lib/test_config.ml

```diff
@@ -18,6 +18,15 @@ module Test_Account = struct
     }
 end
 
+module Epoch_data = struct
+  module Data = struct
+    (* the seed is a field value in Base58Check format *)
+    type t = { epoch_ledger : Test_Account.t list; epoch_seed : string }
+  end
+
+  type t = { staking : Data.t; next : Data.t option }
```

Comment:
why `next` is an option? Is it because that we would only have a hard fork if when the network pass the first epoch and we would at least have a staking epoch ledger?

Reply:
This is mainly for if HF occurs in the first epoch (or subsequent ones until there's a new snarked ledger) but probably won't happen for mainnet, unless for some reason, the next ledger is equal to staking at fork. Let's keep it as it.